include/wil/common.h

```diff
@@ -261,7 +261,7 @@ WI_ODR_PRAGMA("WIL_KERNEL_MODE", "1")
 WI_ODR_PRAGMA("WIL_KERNEL_MODE", "0")
 #endif
 
-#if defined(_CPPUNWIND) && !defined(WIL_SUPPRESS_EXCEPTIONS)
+#if defined(_CPPUNWIND) || defined(__EXCEPTIONS) && !defined(WIL_SUPPRESS_EXCEPTIONS)

```

Comment:
Parentheses?

Reply:
Yeah, this breaks WIL_SUPPRESS_EXCEPTIONS on MSVC

Reply:
I'll fix it soon.